lib/anagrams.rb

```diff
@@ -0,0 +1,28 @@
+class Anagrams
+  RESULT = {}
+
+  def initialize(filename)
+    @filename = filename
+  end
+
+  def read
+    File.open(@filename).read.split("\n")
+  end
+
+  def get_anagrams
+    read.each do |word|
+      key = word.split('').sort.join
+      if RESULT.key?(key)
+        RESULT[key].push(word)
+      else
+        RESULT[key] = [word]
+      end
+    end
+    RESULT.values
```

Comment:
@StefanGajic the usage of `RESULT` is a bit weird. It's kind of caching the results, but you have to go through all words each time regardless of the fact that you might have already called it.